### PR TITLE
switch to dzil

### PR DESCRIPTION
**Pull request by chylli**

### CODE REVIEW

File: dist.ini [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+[PodSyntaxTests]
+[MojibakeTests]
+[Test::CheckDeps]
+[Test::Compile]
+[Test::Synopsis]
+[Test::EOL]
+[Test::Version]
+[Test::Pod::LinkCheck]
+[PodCoverageTests]
+;[Test::UnusedVars]
```
Comment (chylli) [inline dist.ini]: There is a bug in this plugin for module 'Concise.pm'. So I disabled it. Please see the comment in `Concise.pm`

File: lib/Time/Duration/Concise.pm [hunk 2]
```diff
@@ -207,8 +203,9 @@ sub as_concise_string {
     $precision ||= 10;
     my $time_frames         = $self->_duration_array($precision);
     my @concise_time_frames = map {
-        $_ =~ s/\s+//ig;
```
Comment (chylli) [inline lib/Time/Duration/Concise.pm]: `$_` should not be modified in the list according to `perlcritic` policy.
So I added a variable `$t`. But there is a bug in `Dist::Zilla::Plugin::Test::UnusedVars` (https://github.com/doherty/Dist-Zilla-Plugin-Test-UnusedVars/issues/7)

So I disabled this plugin in `dist.ini`

Comment (basiliscos) [inline lib/Time/Duration/Concise.pm]: coud you try just: 
```
s/\s+//gr
```

? the `$_ =~` is done explicitely, and `r` flag means return modified value and keep original untouched, rather then  modify original. Aslo, we don't need `i` here (ignore case)

Comment (chylli) [inline lib/Time/Duration/Concise.pm]: Sorry I don't understand.
Do you mean the following code ?
```
map {
   my $t = s/\s+//rg;
   $t =~  /([-|\+]?\d+[A-Za-z]{1})/ig;
  $1;
} @$time_frames;
```

It still not work for `UnusedVars`

Comment (basiliscos) [inline lib/Time/Duration/Concise.pm]: OK, but it is a bit shorter :)

Comment (chylli) [inline lib/Time/Duration/Concise.pm]: @basiliscos Changed and OK now.  Could you please review the diff again ?

**Pull request merged**